.github/workflows/release.yml

```
@@ -49,20 +49,60 @@ jobs:
           asset_name: evm-indexer-v${{ github.event.release.tag_name }}-x86_64-unknown-linux-gnu.tar.gz
           asset_content_type: application/gzip
 
+  docker:
+      name: Build and Push Docker Image
```

yamllint

[error] wrong indentation: expected 4 but found 6

(indentation)